mobile.html

```diff
@@ -156,5 +156,16 @@ <h5>2015</h5>
             <li><img src="./images/icons/Separator.png" alt="sep"></li>
         </ul>
     </section>
+    <div class="form-section">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class ="form" action="https://formspree.io/f/mgedlyee" method="post">
+            <input class="input1" type="text" placeholder="Yeremias" maxlength="30">
+            <input class="input1" type="email" placeholder="natajayanj14@gmail.com">
+            <textarea class="message" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] All input fields have the `required` attribute to activate the validation on submit. Let's add `required in every input.